FAQ.md

```diff
@@ -87,8 +87,13 @@ When migrating to a new phone, just install Aegis again, and the Cloud provider
 will restore its backup to the new device. You can test restoring a backup by
 uninstalling the app and re-installing it. To convince yourself that the data
 wasn't just left on the device, you can _Clear storage_ and _Clear cache_
-in Aegis' App info _Storage and cache_ settings before uninstalling it. If
-you recently added a new entry, then before doing such a test, make sure
+in Aegis' App info _Storage and cache_ settings before uninstalling it.
+
+The password that it asks you for after a (re)install means that the restore
```

Comment:
```suggestion
If Aegis is already installed and you see the welcome screen when starting the app 
although you were expecting to restore your backup, reinstall the app. This should 
help to get you into the restore process from above, asking you for your password.

The password that it asks you for after a (re)install means that the restore
```

These are the findings I described in https://github.com/beemdevelopment/Aegis/pull/1714/files/e0b68e1575fbb5903897f8533ef5269d1ea59448..6be993fc885bf48272c921301eec7845bd483018#r2440755457. Maybe this should also become a bug ticket?